src/Netscript/RamCostGenerator.ts

```diff
@@ -355,6 +359,8 @@ const grafting = {
 const corporation = {
   hasCorporation: 0, // This one is free
   getConstants: 0,
+  getBonusTime: 0,
+  nextCycle: RamCostConstants.CycleTiming,
```

Comment:
I think this is the wrong cost, since the API it would replace (getCorporation.getState()) has a cost of CorporationInfo.

(I think that will be too expensive, but I think *all* the corp costs are too expensive, so at least this is consistent.)

Reply:
I reckon this approach is reasonable as it is similar to the situation with `ns.getServer()` for 2 GB vs the various `ns.getServer___()` functions for 0.1 GB each. It adds some new tradeoffs to consider when designing small single-purpose corp scripts, but it does not break any existing APIs.